src/lib.rs

```diff
@@ -90,7 +96,8 @@ impl From<&str> for KolorScheme {
             "catppuccin mocha" => Self::CatppuccinMocha,
             "dracula" => Self::Dracula,
             "nord" => Self::Nord,
-            "gruvbox" => Self::Gruvbox,
+            "gruvbox dark" => Self::GruvboxDark,
+            "gruvbox light" => Self::GruvboxLight,
```

Comment:
Looks like you're missing the branch for biscuit dark and biscuit light

Reply:
I see, my bad! Thank you